.github/workflows/integration.yml

```diff
@@ -67,17 +65,6 @@ jobs:
           go build -o ${INSTALL_PATH}/guardian cmd/guardian/main.go
           echo "${INSTALL_PATH}" >> $GITHUB_PATH
 
-      - name: 'Guardian Directories'
```

Comment:
I think we want to keep this, only remove the detect changes. So the syntax should be:

```
          DIRECTORIES=$(guardian entrypoints -format=json ./.)
          echo "entrypoints -> ${DIRECTORIES}"
          echo "directories=${DIRECTORIES}" >> $GITHUB_OUTPUT
```

Reply:
This way we "test" the entrypoints also

Reply:
done.

Reply:
Hmm with this change, directories from `pkg` are not being included

Reply:
It should be targeted at `./terraform`, not sure why this differs from the template: 

https://github.com/abcxyz/guardian/blob/main/abc.templates/base-workflows/contents/guardian-plan.yml#L75

Reply:
Yes, I've updated it